src/org/parosproxy/paros/network/SSLConnector.java

```diff
@@ -673,11 +677,11 @@ private void createX509KeyManager(Socket socket) {
 			String hostname = extractHostname(sslSocket.getHandshakeSession());
 
 			if (hostname == null) {
-				logAndThrow("No domain extracted from SSL/TLS handshake session.");
+				logger.info("No domain extracted from SSL/TLS handshake session.");

```

Comment:
This could be debug no longer critical (we will validate/fail later with a more relevant message).